Directory.Build.props.todo

```diff
@@ -0,0 +1,8 @@
+<Project>
+	<PropertyGroup>
+		<!-- Change this to the path of your modded HK installation -->
```

Comment:
Defaulting this to `C:\Program Files (x86)\Steam\steamapps\common\Hollow Knight\hollow_knight_Data\Managed` (HK's default install location on a default Steam install) might not be awful versus leaving it blank.

I nuked my previous setting and failed to remember it also needs the `hollow_knight_data\Managed` portion, so updating the comment might be useful (though a semi-valid path as an exampe might be enough documentation).